pkg/lib/config/testdata/default_config.yaml

```diff
@@ -270,6 +270,9 @@ identity:
         case_sensitive: false
         block_plus_sign: false
         ignore_dot_sign: false
+        domain_blacklist_enabled: false
+        domain_whitelist_enabled: false
+        include_default_blacklisted_domains: false
```

Comment:
as mentioned above, "default blacklisted domains" is not clear.

Reply:
Let me change it to `block_free_email_provider_domains: false`